meta/src/grammar.pest

```diff
@@ -16,102 +16,126 @@
 //! future (e.g. by increasing MSRV and non_exhaustive annotations).
 
 /// The top-level rule of a grammar.
-grammar_rules = _{ SOI ~ grammar_doc* ~ (grammar_rule)+ ~ EOI }
+grammar_rules = _{ SOI ~ grammar_doc* ~ grammar_rule* ~ EOI }
```

Comment:
This first just changed this line from `(grammar_rule)+` to `grammar_rule*`.

The other changes was made by pest-fmt format.